glusterd2/utils/pprof.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"net/http"
+	"net/http/pprof"
+	runtime_pprof "runtime/pprof"
+
+	"github.com/gorilla/mux"
+)
+
+var (
+	basePath string = "/debug/pprof/"
+)
+
+func addPath(router *mux.Router, name string, handler http.Handler) {
+	router.Path(basePath + name).Name(name).Handler(handler)
```

Comment:
add a debug log to know what are the registered profiling handlers.

Reply:
done